### DISCUSSION TITLE
skip file and continue if error occurs

### DISCUSSION THREAD

**Discussion by Unwinter** [nagadomi/nunif#133] [Ideas]
I think a "skip file if error occurs" would be great. For me at least. I want to go to work and not babysit a batch of files and just handle the errors when the batch is finished like with a log of filenames or something. Unless its an out of free space error. I know there is a big list of wishes, just adding my 5 cents.

**Comment by nagadomi** [comment]
This is related to some issues, but I will try to support the "skip error" option sometime.

As a problem related to this,
there is a bug that causes CUDA out of memory error if the image aspect size is very different (e.g. 16x1000). I plan to fix that bug.

For corrupted image files, they should be silently skipped currently.
